.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
Shame flake8 doesn't like `pyproject.toml` natively (yet).

As we're specifying the file path in the pre-commit with `args: ["--config=.flake8"]` - does it work if we just move this to the `pyproject.toml` file?

Reply:
Unfortunately it does not.
There are packages now that do support this but given they are stil in alpha testing, I wouldn't go for it.
E.g. https://pypi.org/project/pyproject-flake8/

Reply:
Ok, thanks for checking 👍🏻